test/rules/member-ordering/alphabetize/test.ts.fix

```diff
@@ -3,6 +3,16 @@ class X {
     bar: number;
     foo: string;
 
+    get y() {
+        return 0;
+    }
+    set y(x: number) {}
+
+    set z(x: number) {}
+    get z() {
```

Comment:
why does it go `get set set get`? i would expect `get set get set`.

Reply:
This was discussed in the underlying issue https://github.com/palantir/tslint/issues/3965

Currently the sort works by first sorting by category, then optionally sorting alphabetically. To get 

    get x
    set x
    get y
    set y
    // other members

would require sorting first by category, then by alphabet, and a third "pass" sorting get < set. I've left this out as it would be added complexity (maybe some people want set < get) for what would only be very local errors. 

Adding this it isn't out of the question if you feel it's important.

Reply:
can we interpret these as `x get, `x set`, `y get`, `y set`? that'll give the desired sorting.

Reply:
Heh, I had the exact same idea https://github.com/palantir/tslint/issues/3965#issuecomment-398552202. I dismissed it would result in things like this:
```
get x() {}
"x lol"() {}
set x() {}
```

Rethinking it however, that seems like such an unlikely scenario that it might still be a good solution.